.idea/runConfigurations/Client.xml

```diff
@@ -3,15 +3,17 @@
     <extension name="coverage" enabled="false" merge="false" sample_coverage="true" runner="idea" />
     <option name="MAIN_CLASS_NAME" value="org.exist.client.InteractiveClient" />
     <option name="VM_PARAMETERS" value="" />
-    <option name="PROGRAM_PARAMETERS" value="" />
+    <option name="PROGRAM_PARAMETERS" value="-l" />
```

Comment:
Did you really mean to set the client to embedded mode by default as well in this commit? If you did, it suits me :-)